java-numbers-3/pom.xml

```diff
@@ -1,38 +1,58 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>java-numbers-3</artifactId>
-    <name>java-numbers-3</name>
-    <packaging>jar</packaging>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>java-numbers-3</artifactId>
+	<name>java-numbers-3</name>
+	<packaging>jar</packaging>
 
-    <parent>
```

Comment:
Let's return to indentation with spaces and avoid making whitespace-only changes.